AppController/lib/backup_restore_service.rb

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/ruby -w
+
+
+$:.unshift File.join(File.dirname(__FILE__))
+require 'helperfunctions'
+require 'monit_interface'
+
+
+# Starts and stops the backup and recovery service.
+module BackupRecoveryService
+
+  BR_PORT = '8423'
+
+  # Starts the BR Service on this machine. We don't want to monitor
+  # it ourselves, so just tell monit to start it and watch it.
+  def self.start()
+    start_cmd = "/usr/bin/python /root/appscale/AppDB/backup/backup_recovery_service.py"
+    # The stop command doesn't work and relies on terminate.rb.
```

Comment:
I don't understand this comment. Is it just because the the start method requires a stop_cmd argument?